### PR TITLE
Replaced Cassandra-uuid with uuid

### PR DESCRIPTION
**Pull request by hknustwmf**
Bug: T223690

### CODE REVIEW

**Comment by d00rman** [comment]
@hknustwmf the linter complains about spaces around brackets. Let's disable that rule, as having `( [ ... ] )` is not consistent with the way we format other parentheses (it should be `([ ... ])`).

File: lib/util.js [hunk 1]
```diff
@@ -91,9 +91,13 @@ function errForLog(err) {
  * @return {!string} the generated request ID
  */
 function generateRequestId() {
-
-    return uuid.TimeUuid.now().toString();
-
+    const options = {
```
Comment (d00rman) [inline lib/util.js]: None of these options are actually needed. Le'ts remove them.

**Review by d00rman — CHANGES_REQUESTED** [review]

File: lib/util.js [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 const BBPromise = require('bluebird');
 const express = require('express');
-const uuid = require('cassandra-uuid');
+const uuidV1 = require('uuid/v1');
```
Comment (d00rman) [inline lib/util.js]: Let's use all lowercases here, i.e. `uuidv1`

**Review by d00rman — DISMISSED** [review]

File: lib/util.js (hunk 1 above)
Comment (hknustwmf) [reply lib/util.js]: Removed the options

**Comment by hknustwmf** [comment]
> @hknustwmf the linter complains about spaces around brackets. Let's disable that rule, as having `( [ ... ] )` is not consistent with the way we format other parentheses (it should be `([ ... ])`).

I turned array-bracket-spacing off

File: lib/util.js (hunk 2 above)
Comment (hknustwmf) [reply lib/util.js]: Changed to lowercase

**Review by Pchelolo — APPROVED** [review]

**Pull request merged**